fyrox-ui/src/inspector/mod.rs

```diff
@@ -1025,6 +1021,44 @@ impl Control for Inspector {
             }
         }
 
+        if !message.handled() {
+            if let Some(WidgetMessage::MouseDown { button, .. }) = message.data() {
+                if *button == MouseButton::Right && self.context.menu.menu.is_some() {
+                    message.set_handled(true);
+                    let menu: &Menu = &self.context.menu;
+                    menu.target.set(message.destination());
+                    let menu_handle = menu.menu.clone().unwrap().handle();
+                    ui.send_message(PopupMessage::placement(
+                        menu_handle,
+                        MessageDirection::ToWidget,
+                        Placement::Cursor(message.destination()),
+                    ));
+                    ui.send_message(PopupMessage::open(menu_handle, MessageDirection::ToWidget));
+                }
+            }
+        }
+
+        // Check for context menu item clicked. We get this message because we are the owner of the popup that contains the menu.
+        if message.destination() == self.context.menu.copy_value_as_string {
+            if let Some(MenuItemMessage::Click) = message.data() {
+                let mut target = self.context.menu.target.get();
+
+                while let Some(node) = ui.try_get(target) {
+                    for entry in self.context.entries.iter() {
+                        if entry.property_container == target {
```

Comment:
This starts at the target from the Menu struct and crawls up the widget tree in just the same way, stopping when it finds the container for an editor widget or None. The only difference is using `self.context.menu.target.get()` as the start of the crawl instead of using a hit_test as the start of the crawl.

Reply:
Ah, yes, sorry, didn't see `target = node.parent;` line.